ERCS/erc-xxx.md

```diff
@@ -0,0 +1,321 @@
+---
```

Comment:
```suggestion
---
eip: 7570
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.